config.json

```diff
@@ -1,4 +1,4 @@
 {
   "prefixRedaction": "/rvs",
   "prefixEditor": "/edt"
-}
+}
```

Comment:
Why do you removed the blank line?

Reply:
For no reason, I believe I deleted it without realizing it or some wrong config in my editor.